pkg/skaffold/build/cache/hash.go

```diff
@@ -66,14 +67,19 @@ func newArtifactHasher(artifacts graph.ArtifactGraph, lister DependencyLister, m
 }
 
 func (h *artifactHasherImpl) hash(ctx context.Context, a *latestV1.Artifact) (string, error) {
+	ctx, endTrace := instrumentation.StartTrace(ctx, "hash_GenerateHashOneArtifact")
```

Comment:
This seems like a good place to include the image name
```suggestion
	ctx, endTrace := instrumentation.StartTrace(ctx, "hash_GenerateHashOneArtifact", map[string]string{
		"ImageName": instrumentation.PII(a.ImageName),
	}))
```

Reply:
Done